### PR TITLE
[Docs] Fix incorrect Javascript term in AutocompleteInput docs

### PR DESCRIPTION
**Pull request by davidhenley**
These are Javascript objects, not tuples

### CODE REVIEW

**Review by slax57 — COMMENTED** [review]
Thanks for submitting this anyway :slightly_smiling_face:

File: docs/AutocompleteInput.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ It renders using MUI [Autocomplete](https://mui.com/components/autocomplete/).
 
 ![AutocompleteInput](./img/autocomplete-input.gif)
 
-Set the `choices` attribute to determine the options list (with `id`, `name` tuples).
+Set the `choices` attribute to determine the options list (with `id`, `name` objects).
```
[the remark this answers is not in the document]
Comment (davidhenley) [reply docs/AutocompleteInput.md:13]: I mean, they are javascript objects with id and name properties. I agree, the wording is a bit weird.

[the remark this answers is not in the document]
Comment (fzaninotto) [reply docs/AutocompleteInput.md:13]: I suggest

> Set the `choices` attribute to determine the options list. The value must be an array of objects exposing the `id` and `name` for each choice.

**Comment by fzaninotto** [comment]
Superseded by #7949

**Pull request closed without merging**